.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -24,6 +24,10 @@ labels: 'bug'
 <!-- A clear and concise description of what actually happened. -->
 
 
+### Screenshot
+<!-- Screenshot of the part where the bug occurs -->
+
+
```

Comment:
With a dedicated section for screenshots, I think we can remove the mention of them in the "Additional Information" section. (Or just remove that section altogether)